config/webpacker.yml

```diff
@@ -5,6 +5,7 @@ default: &default
   source_entry_path: packs
   public_output_path: packs
   cache_path: tmp/cache/webpacker
+  compile: true
```

Comment:
I don't think we want this - Heroku automatically compiles assets using `rails assets:precompile`, can we revert?